thirdparty/erlang/patches/otp-0003-Do-not-install-erlang-sources.patch

```diff
@@ -0,0 +1,336 @@
+diff --ignore-blank-lines --ignore-space-change -u -r otp-24.0/lib/asn1/src/Makefile otp-24.0-patched/lib/asn1/src/Makefile
+--- otp-24.0/lib/asn1/src/Makefile	2021-05-12 21:31:53.000000000 +0100
++++ otp-24.0-patched/lib/asn1/src/Makefile	2020-05-13 23:51:34.000000000 +0100
```

Comment:
why are we removing erlang sources?

Reply:
we are creating zero dependency erlang package that enough to tun rabbitmq